xml/System.Text.Json/Utf8JsonWriter.xml

```diff
@@ -383,7 +383,7 @@ The <xref:System.Text.Json.Utf8JsonWriter> will continue to use the original wri
 
 ## Remarks
 
-The <xref:System.Text.Json.Utf8JsonWriter> will continue to use the original writer options but now write to `bufferWriter` as the new destination.
+The <xref:System.Text.Json.Utf8JsonWriter> will continue to use the original writer options but now writes to `bufferWriter` as the new destination.
```

Comment:
Re-reading https://github.com/dotnet/dotnet-api-docs/pull/3143#discussion_r322547543, looks like this should either be "write" or "will write".

Reply:
Let's wait for @rpetrusha to review this one. He said one thing on the comment and another thing via text message 😉